src/tests/sentry-mocked.test.ts

```diff
@@ -94,7 +94,6 @@ describe('sentry mocked', () => {
                   "level": 60,
                   "message": "fatal",
                   "time": "2018-03-06T13:30:36.000Z",
-                  "v": 1,
```

Comment:
Why has this changed @vlasy ?

Reply:
It's one of the pino's changes: https://github.com/pinojs/pino/pull/623

Reply:
Thanks